content/posts/hanami-repository-bulk-update.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Hanami Repository Bulk Update"
+preview: "Update multiple database records at once"
+status: "Learn how to efficiently update a collection of records"
+date: 2018-10-30T15:45:16+01:00
+draft: true
```

Comment:
@kaikuchn Please don't forget to set this to `false`. We should probably remove it from the code generator.